src/test/resources/pages/newlyJoinedInstructorFeedbackSessionSuccessEdited.html

```diff
@@ -213,7 +213,7 @@ <h5 class="col-sm-2 margin-top-0">
                 <div class="col-sm-10">
                   <div class="panel panel-default panel-body mce-content-body content-editor" id="instructions" spellcheck="false">
                     <p>
-                      updated instructions
+                      Please give your feedback based on the following questions.
```

Comment:
You need to re-run the test again with GodMode switched off to ensure it works. More details why this is so can be found in the [GodMode doc](https://github.com/TEAMMATES/teammates/blob/master/docs/godmode.md#final-notes).